build.ps1

```diff
@@ -19,6 +19,10 @@ if ($env:IntegrationBuildNumber)
 
     $buildCommand = "dotnet run --integrationTests"
 }
+elseif ($env:IsReleaseBuild)
```

Comment:
Do we only want to include a SBOM for release builds?

Reply:
Yes, we will only be including an SBOM for release builds.